### ISSUE TITLE
About benchmark results

### ISSUE THREAD

**Issue by 1787648106** [pytorch-labs/gpt-fast#19]
Amaizing work! How was the benchmark results obtained? Is it just the generation speed measured when the GPU power limited 330W?

**Comment by Chillee** [comment]
Yes. GPU power limit is an unfortunate limitation of the particular hardware setup I'm using - it's not required.
